.github/workflows/release.yml

```diff
@@ -1,111 +1,56 @@
-name: release Boltzmann
+name: build Boltzmann for release
 on:
-  push:
-    tags:
-      - 'v*'
-
+  release:
+    types: [published]
 jobs:
   release:
-    name: release
+    name: publish npm package
```

Comment:
This builds a "fat binary", so to speak, including all platforms and architectures for boltzmann in a single npm package. The javascript included selects the correct binary using `os.arch()` + `os.platform()`, hence the change from `x86` to `x64` above.